homeassistant/components/yale_smart_alarm/strings.json

```diff
@@ -1,28 +1,43 @@
 {
   "config": {
     "abort": {
-      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
+      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
```

Comment:
Bugfix, not related to this PR.

Reply:
Will be taken care off in new PR

Reply:
This was missing in the strings.json file from previous PR but was part of the config flow.
So I hope we can leave it in and I do the remaining bugfixes in separate PR